lib/modules/datasource/azure-tags/schema.ts

```diff
@@ -0,0 +1,4 @@
+export interface AzureTag {
```

Comment:
use zod schema

Reply:
Belongs to the other comment. Does not work if I use unchecked with PagedResult